openpecha/core/metadata.py

```diff
@@ -115,3 +115,42 @@ class DiplomaticPechaMetadata(PechaMetadata):
     @validator("id", pre=True, always=True)
     def set_id(cls, v):
         return v or ids.get_diplomatic_id()
+
+class AlignmentMetadata(BaseModel):
+    id:str
+    title:str
+    type: str
+    pechas: List[str]
+    alignment_to_base:Dict[str,str]
+    source_metadata: Optional[Dict] = None
+
+class InstanceMetadata(BaseModel):
```

Comment:
can you document a little bit what an instance, an alignment, a work and a collection are? My intuition is that instances should not have authors but should instead have works, but since it's all a bit unclear I'm not sure